README.md

```diff
@@ -25,10 +39,14 @@ nix = "0.6.0"
 
 Then, add this to your crate root:
 
-```rust
+```rust,ignore
```

Comment:
I didn't add it yet, but was planning on adding support for [skeptic](https://github.com/brson/rust-skeptic)